client/state/posts/actions.js

```diff
@@ -68,3 +71,36 @@ export function requestSitePosts( siteId, query = {} ) {
 		} );
 	};
 }
+
+/**
+ * Triggers a network request to fetch a specific post from a site.
+ *
+ * @param  {Number}   siteId Site ID
+ * @param  {Number}   postId Post ID
+ * @return {Function}        Action thunk
+ */
+export function requestSitePost( siteId, postId ) {
+	return ( dispatch ) => {
+		dispatch( {
+			type: POST_REQUEST,
+			siteId,
+			postId
+		} );
+
+		return wpcom.site( siteId ).post( postId ).get().then( ( post ) => {
+			dispatch( receivePost( post ) );
+			dispatch( {
+				type: POST_REQUEST_SUCCESS,
+				siteId,
+				postId
+			} );
```

Comment:
>Promises always fulfill or reject on the next (well, a future) turn of the event loop, unless you mean something else?

Yeah, I mean something else. Naive attempts by me in testing nock would look more like:

```jsx
it( 'should finish request on next tick', ( done ) => {
	someRequestMockedByNock();

	setTimeout( () => {
		expect( myAsyncSpy ).to.have.been.calledOnce;
		done();
	}, 0 );
} );
```

But for whatever reason, the nock mocks aren't resolved in any predictable next tick.